ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/InternalPaywall.kt

```diff
@@ -33,16 +36,20 @@ internal fun InternalPaywall(
 ) {
     PaywallTheme(fontProvider = options.fontProvider) {
         viewModel.refreshStateIfLocaleChanged()
-        val colors = MaterialTheme.colorScheme
-        viewModel.refreshStateIfColorsChanged(colors)
+        viewModel.refreshStateIfColorsChanged(MaterialTheme.colorScheme)
 
         when (val state = viewModel.state.collectAsState().value) {
             is PaywallState.Loading -> {
                 LoadingPaywall(mode = options.mode)
             }
 
             is PaywallState.Error -> {
-                Text(text = "Error: ${state.errorMessage}")
+                LoadingPaywall(mode = options.mode)
```

Comment:
Displaying this so that when loading fails we don't make the content disappears.